tests/test_extract.py

```diff
@@ -34,6 +32,9 @@ class LoginResponse(BaseModel):
         password: str = Field(..., description="The password.")
         system_prompt: ClassVar[str] = "Extract the username and password from the input."
 
+        class Config:
+            extra = "forbid"
+
     username, password = "cypher", "steak"
     login_response = extract_with_llm(LoginResponse, f"{username} // {password}", config=config)
```

Comment:
After applying the suggestion above:
```suggestion
    login_response = extract_with_llm(LoginResponse, f"{username} // {password}", strict=True, config=config)
```